go.mod

```diff
@@ -4,17 +4,16 @@ go 1.12
 
 require (
 	github.com/gobuffalo/flect v0.1.5
-	github.com/inconshreveable/mousetrap v1.0.0 // indirect
 	github.com/kr/pretty v0.1.0 // indirect
 	github.com/onsi/ginkgo v1.8.0
 	github.com/onsi/gomega v1.5.0
 	github.com/spf13/afero v1.2.2
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/pflag v1.0.3
-	golang.org/x/net v0.0.0-20190514140710-3ec191127204 // indirect
 	golang.org/x/sys v0.0.0-20190506115046-ca7f33d4116e // indirect
 	golang.org/x/text v0.3.2 // indirect
 	golang.org/x/tools v0.0.0-20190506145303-2d16b83fe98c
 	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127 // indirect
 	gopkg.in/yaml.v2 v2.2.2
+	sigs.k8s.io/kind v0.4.0 // indirect
```

Comment:
Please drop this. I believe this is caused by running `go get sigs.k8s.io/kind` in this repo.
And also please run `go mod tidy` to ensure `go.sum` only have necessary deps.

Reply:
done.